src/main/resources/liquibase/crop_changelog/19_4_0.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="cuenyad" id="v19.4.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cvterm" columnName="is_system"/>
+			</not>
+		</preConditions>
+		<comment>Adding is_system_term column</comment>
+        <sql>
+			ALTER TABLE cvterm ADD COLUMN is_system INT(11) NOT NULL DEFAULT '0' AFTER is_relationshiptype;
+        </sql>
+    </changeSet>
+
+	<changeSet author="cuenyad" id="v19.4.0-2">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">SELECT count(1) > 0 FROM cvterm where cvterm_id &lt;= 100000 and cv_id != 1040 and is_system = 0</sqlCheck>
+		</preConditions>
+		<comment>Mark system variables</comment>
+        <sql>
+			update cvterm cv INNER JOIN cvterm cv1 ON cv.cvterm_id = cv1.cvterm_id  set cv.is_system = 1
+			where cv1.name in ('TRIAL_INSTANCE','LOCATION_ID','LOCATION_NAME','EXPT_DESIGN','FIELDMAP COLUMN'
```

Comment:
Another possibility, given that it could be difficult to find just one query that matches all of what we want to mark as _system variable_, is to use a generic query to mark the _majority_ of the terms we want, and then use a separate update to mark the _edge cases_ that didn't fall in the first criteria